### DISCUSSION TITLE
How to prevent the window (not tab) title from changing

### DISCUSSION THREAD

**Discussion by shaggyaxe** [microsoft/terminal#17938] [Q&A] [answered]
O/S version: `Microsoft Windows [Version 10.0.22631.4169]`
Terminal version: `1.21.2361.0`

I'd like to start `wt` with an arbitrary title, and have it never change.

e.g., 
`wt --title foo`

I know I can do `"suppressApplicationTitle": true` to prevent the `title` command from changing it. 
But the window title can be changed later by things like:
- Renaming the current tab
- Opening a new tab
  - e.g., opening a PowerShell tab will change the window title to "PowerShell"

I just want the title of the window to remain the initial title, and have it never change.  Regardless of the titles of the tabs.

Thanks

**Comment by DHowett** [comment, marked as answer]
That’s definitely something we haven’t fleshed out - giving the application window a title other than “Windows Terminal” but not putting the console application in control of it. There isn’t a way to do this today.

**Comment by gkTim** [comment]
Was the hide title bar removed or is it simply not available on mac?

**Reply by DHowett** [reply]
> available on mac

Sorry? This is the repository for Windows Terminal. Our software does not run on macOS.
